apps/demo/translations/src/en/fields.json

```diff
@@ -3,13 +3,16 @@
     "legend": "Which form would you like to explore?",
     "options": {
       "basic-form": {
-        "label": "Basic form"
+        "label": "Basic form",
+        "hint": "An example of a simple form produced using the hof framework. This form demos common components used by services using the hof framework."
       },
       "complex-form": {
-        "label": "Complex form"
+        "label": "Complex form",
+        "hint": "An example of a form produced using the hof framework. This longer demo includes some steps from the basic form and features less common components used in other services using hof."
       },
```

Comment:
I'd maybe change this to say that it "showcases more advanced features available in the HOF framework" as well.

Reply:
Made the suggested changes to the second sentence.